aspnetcore/fundamentals/minimal-apis.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Minimal APIs overview
+title: Minimal APIs cheat sheet 
```

Comment:
Consider what we have for System.Text.Json for comparison -- 

* An [Overview](https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/overview) that gives a high-level description of features and when/why to use it. 
* A main how-to doc like the single cheat sheet for Minimal APIs - "[How to serialize and deserialize JSON in .NET](https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/how-to)". (It got too big and we extracted parts of it into other articles, but it started out as the one STJ cheat sheet.)

So expanding on @Rick-Anderson's suggestion, we could name this "How to create and configure Minimal API apps."

Reply:
I thought we agreed to call it a "Cheat sheet" instead of an "overview" because this is far more in-depth and comprehensive. Is that not what we agreed to?

I'm fine with reverting this, but I don't want to go back and forth on this. I want to get this merged ASAP because this is a major refactoring that is likely to conflict with work others are doing in this area. @JeremyLikness @samsp-msft @davidfowl

Reply:
So the suggested options so far are:

```suggestion
title: Minimal APIs overview
```

```suggestion
title: Minimal APIs quick overview
```

```suggestion
title: Minimal APIs quick reference
```

```suggestion
title: Minimal APIs how to
```

```suggestion
title: How to create and configure Minimal API apps
```

```suggestion
title: Minimal APIs how to
```

I prefer "cheat sheet", but I'm happy to switch it to whatever else has consensus. I just want an answer for which one I should switch to. I'll update all the other references to the title to match.

Reply:
Assuming "cheat sheet" is an absolute no go, I think "quick reference" is my second favorite option.

Reply:
> Assuming "cheat sheet" is an absolute no go, I think "quick reference" is my second favorite option.

Definitely not an absolute `no go`, but slang is strongly discouraged. 

`quick reference` is a great alternative that readily MTs and doesn't loose much for those familiar with English slang.

Reply:
I like quick reference